raml-parser-2/src/main/java/org/raml/v2/internal/impl/commons/nodes/ExternalSchemaTypeExpressionNode.java

```diff
@@ -47,11 +46,7 @@ public String getSchemaValue()
 
     public String getSchemaPath()
     {
-        if (StringUtils.isNotBlank(this.getStartPosition().getIncludedResourceUri()))
-        {
-            return this.getStartPosition().getIncludedResourceUri();
-        }
-        return this.getStartPosition().getPath();
+        return this.getStartPosition().getIncludedResourcePath();
```

Comment:
I think returning just `getIncludedResourcePath()` we are missing the case where the schema is not included but is defined inline